syntax = "proto3";

import "SignatureMessage.proto";
import "Empty.proto";

package proto;

service RemainService {
  rpc onSignature(SignatureMessage) returns (Empty) {}
}
